conf4j-core/src/main/java/com/sabre/oss/conf4j/converter/standard/BooleanTypeConverter.java

```diff
@@ -30,14 +30,32 @@
 import java.util.Map;
 import java.util.Objects;
 
+import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
+import static org.apache.commons.lang3.BooleanUtils.*;
 
 /**
- * This class converts {@link Boolean} to/from string.
+ * <p>This class converts {@link Boolean} to/from string.</p>
+ * <p>The converter supports {@code format} attribute (provided in the attributes map) which specifies
+ * the format used during conversion.</p>
+ * <p>Possible formats:</p>
+ * <ul>
+ * <li><b>string</b> - returns "true" or "false"</li>
+ * <li><b>stringYesNo</b> - returns "yes" or "no"</li>
+ * <li><b>stringOnOff</b> - returns "on" or "off"</li>
+ * <li><b>integer</b> - returns "1" or "0"</li>
+ * </ul>
+ * <p>
+ * When the format is not specified, {@link Objects#toString() } method is used.
  */
 public class BooleanTypeConverter implements TypeConverter<Boolean> {
```

Comment:
I would suggest introducing _format_ attribute which value are `/` (slash) separated literals e.g. `false/true`, `on/off` etc. It will be simpler and more customizable the current solution.